src/twisted/names/secondary.py

```diff
@@ -107,10 +107,10 @@ class SecondaryAuthority(FileAuthority):
 
     @ivar _port: The port number of the server from which zone transfers will
     be attempted.
-    @type: C{int}
+    @type _port: C{int}
 
     @ivar domain: The domain for which this is the secondary authority.
-    @type: C{bytes}
```

Comment:
From the https://twistedmatrix.com/documents/current/core/development/policy/coding-standard.html#docstrings

```
When you are referring to a type, you should use L{}, whether it’s in the stdlib , in Twisted or somewhere else.
```

So this needs to be `L{bytes}`.

Reply:
I'll fix it.

There are hundreds of other cases of `C{typename}` in Twisted's sources though, so maybe it's worth doing a mechanical replace over the whole source tree at some point. Or maybe we can skip that and instead do an automated conversion from `@type` to type annotation/comment.